data/worlds/classic.world

```diff
@@ -75,6 +75,7 @@ overlay
     [ {stone}
     , mask (hash % 10 == 0) {stone, rock}
     , mask (hash % 100 == 0) {stone, lodestone}
+    , mask (hash % 300 == 0) {grass, parsley}
```

Comment:
A stray world test? 🙂

Reply:
No, `parsley` is what provides the `read` capability, so I added some to grow in the world!

Fun English trivia: according to Wikipedia, parsley is native to Greece, and the English word "parsley" ultimately is derived from the Greek "petroselinon" meaning "rock-celery", so-called probably because it tends to grow on cliffs and among rocks.  That's why I added it to the biome with all the rocks.  :rock: :seedling: :rock: :smile: